editor/editor_inspector.cpp

```diff
@@ -2175,7 +2175,7 @@ bool EditorInspectorArray::can_drop_data_fw(const Point2 &p_point, const Variant
 	}
 	Dictionary dict = p_data;
 	int drop_position = _drop_position();
-	if (!dict.has("type") || dict["type"] != "property_array_element" || String(dict["property_array_prefix"]) != array_element_prefix || drop_position < 0) {
+	if (!dict.has("type") || dict["type"] != Variant("property_array_element") || dict["property_array_prefix"] != Variant(array_element_prefix) || drop_position < 0) {
```

Comment:
`dict["property_array_prefix"] != Variant(array_element_prefix)`

should read:
`String(dict["property_array_prefix"]) != String(array_element_prefix)`
perhaps given we explicitly called operator String on LHS previously? Assuming that's valid.

Reply:
Applied freedback